### PR TITLE
Temperature Convertor example

### PR DESCRIPTION
**Pull request by RohitMidha23**
Added a simple example that converts between the two temperature scales.

### CODE REVIEW

File: examples/temperature_convertor/soln.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+option = int(input("Enter choice (1/2): \n1. Fahrenheit to Celsius \n2. Celsius to Fahrenheit \n"))
+
+# based on the option call the appropriate function
+# if the user enters an invalid option print the appropriate response
+if option==1:
+    fahrenheit_to_celsius()
+elif option==2:
+    celsius_to_fahrenheit()
+else:
+    print("INVALID OPTION!")
```
Comment (brahmcapoor) [inline examples/temperature_convertor/soln.py]: Can you stick this in a `main` function and include the `if __name__ == "__main__"` line?

File: examples/temperature_convertor/soln.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
+
+def fahrenheit_to_celsius():
+    # get the fahrenheit temperature from the user
+    fahrenheit = float(input("Enter Temperature in Fahrenheit : "))
+    # use the formula to calculate temperature in celsius
+    celsius = (fahrenheit - 32) * 5.0/9.0
+    # output with appropriate units
+    print("Temperature:", fahrenheit, "F = ", celsius, "C")
```
Comment (brahmcapoor) [inline examples/temperature_convertor/soln.py]: We'll first be teaching printing with string concatenation and explicitly casting ints to strs, like so:

```python
n = 42
print("n = " + str(n)) 
```

so we might want to be consistent with that :)

File: examples/temperature_convertor/soln.py [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+def celsius_to_fahrenheit():
+    # get the celsius temperature from the user
+    celsius = float(input("Enter Temperature in Celsius : "))
+    # use the formula to calculate temperature in fahrenheit
+    fahrenheit = (celsius * 9.0/5.0) + 32
+    # output with appropriate units
+    print("Temperature:", celsius, "C = ", fahrenheit, "F")
```
Comment (brahmcapoor) [inline examples/temperature_convertor/soln.py]: Same thing here

**Review by brahmcapoor — CHANGES_REQUESTED** [review]
Looks great! A few small nitpicks that should be easy to deal with but a cool problem :)

**Comment by RohitMidha23** [comment]
I think that should do it?

File: examples/temperature_convertor/soln.py [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+    # get the celsius temperature from the user
+    celsius = float(input("Enter Temperature in Celsius : "))
+    # use the formula to calculate temperature in fahrenheit
+    fahrenheit = (celsius * 9.0/5.0) + 32
+    # output with appropriate units
+    print("Temperature:" + str(celsius) +  "C = " + str(fahrenheit) + "F")
+
+def main():
+    # get the choice from the user
+    option = int(input("Enter choice (1/2): \n1. Fahrenheit to Celsius \n2. Celsius to Fahrenheit \n"))
```
Comment (brahmcapoor) [inline examples/temperature_convertor/soln.py]: `\n` might be a little confusing, so I might change it to something like this:

```python
print("1.  Fahrenheit to Celsius")
print("2. Celsius to Fahrenheit")
option = int(input("Your choice: ")
```

File: examples/temperature_convertor/index.html [hunk 5]
```diff
@@ -0,0 +1,22 @@
+<p>
+	In 1724, physicist Daniel Gabriel Fahrenheit proposed a temperature scale, the Fahrenheit scale.
+  <br>The Celsius scale, is also a temperature scale but serves as an SI derived unit, and hence used worldwide.
```
Comment (brahmcapoor) [inline examples/temperature_convertor/index.html]: Grammar: change to 'The Celsius scale is also a temperature scale which serves as an SI unit, and hence is used worldwide'

File: examples/temperature_convertor/index.html [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+</p>
+<br>
+<p>
+  Some countries still continue to use the Fahrenheit scale over the Celsius scale.
+  <br>Write a program that takes in a choice from the user, fahrenheit to celsius or vice versa.
+  <br>Read in the appropriate temperature and output the converted temperature.
```
Comment (brahmcapoor) [inline examples/temperature_convertor/index.html:9]: You can probably clarify that the user is guaranteed to type in a real number

**Review by brahmcapoor — COMMENTED** [review]
Awesome, thanks for the quick fix! I noticed three more things that aren't dealbreakers but that you can look into adjusting. Will leave @chrispiech to do the final merge!

**Review by brahmcapoor — APPROVED** [review]

**Pull request merged**